### DISCUSSION TITLE
Transparent cache accleration for existing Object Storage

### DISCUSSION THREAD

**Discussion by sighingnow** [juicedata/juicefs#1976] [General]
Hi folks,

I'm wondering if juicefs support some kind of transparent cache acceleration for object storages? e.g., I have already have a large volume of files on OSS, how could I accelerate the read for such files using juicefs, without downloading them first and the copying those files into the mounted juicefs directory?

Thanks!

**Comment by sighingnow** [comment]
@SandyXSD (polite ping) I would like to know if there are any comments on this topic from the juicefs devs? Thanks!

**Reply by davies** [reply]
Right now, it does not support that, we have a plan to support that after 1.0 released.

**Reply by sighingnow** [reply]
Got it~

**Reply by centminmod** [reply]
> Right now, it does not support that, we have a plan to support that after 1.0 released.

Definitely would like to see that. New to juicefs and using it to play with Cloudflare R2 S3 storage :)
